views/index.erb

```diff
@@ -0,0 +1,14 @@
+<form action="/peeps" method="post">
+  <input type="text" name="peep" placeholder="TYPE YOUR PEEP!" />
+  <input type="submit" value="Submit" />
```

Comment:
Easy to read form, good unambiguous placeholder too so the user knows exactly what input is expected